### DISCUSSION TITLE
Realtiy对于没有域名白名单的地区有必要吗？

### DISCUSSION THREAD

**Discussion by lhbdhr** [XTLS/Xray-core#2274] [Q&A]
如题，目前绝大部分地区应该都没有白名单吧？为什么大家都开始reality？虽然我也是跟风仔，我也用了reality。
但是还是有一点儿困惑，reality对比 [VLESS-TCP-XTLS-Vision](https://github.com/XTLS/Xray-examples/tree/main/VLESS-TCP-XTLS-Vision)有安全方面的优势吗？搭建简单的优势不算，注册个域名也不麻烦。
### 所以我就是想请各位大佬给大家一个建议，什么情况下适合使用reality？
***
对了，补充一个点，而且大部分人还不会设置合适的 reality 的 dest，这反而给 reality 扣分了。

**Comment by RPRX** [comment]
https://github.com/XTLS/Xray-core/issues/2164#issuecomment-1571510589

**Reply by lhbdhr** [reply]
https://github.com/XTLS/Xray-core/issues/2164#issuecomment-1571510589
感谢回复,但是有一个点没懂,不依赖CA什么意思?
不依赖CA的意思就是省了申请TLS证书这一步的另一种说法?
